parkhangs-webapp/src/features/modal/ModalMapDetail.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react'
```

Comment:
this is still not deleted 😬

Reply:
sorry didn't push local changes during class, should be gone :)